EuroPythonBot/registration/cog.py

```diff
@@ -33,20 +33,20 @@ def __init__(self, parent_cog: RegistrationCog):
         self.parent_cog = parent_cog
 
     order_field = discord.ui.TextInput(
-        label="Order ID",
+        label="Order ID (As printed on your badge or ticket)",
         required=True,
         min_length=5,
-        max_length=5,
-        placeholder="123AB",
+        max_length=10,
```

Comment:
```suggestion
        max_length=8,
```
Do we have something longer than `#XXXXX-X` ? if yes, don't mind my suggestion.

Reply:
If an order contains 10 tickets, one of them is `#XXXXX-10`. `max_length=9` is probably enough, though, because orders of 100+ tickets are highly unlikely.

Fixed in 99f144ca.